main.py

```diff
@@ -129,44 +154,10 @@ def respond(voice_data):
             mixer.setvolume(percentage)
 
 
-def PaperScissors(voice_data):
```

Comment:
I think here you have deleted the changed you made in your previous commit 2f817641635afd3604a5b6a750d674d5049076d4 in which you have added PaperScissors() function, an implementation of rock, paper, scissors game.